### DISCUSSION TITLE
E8D83035-EF68-4E18-AB78-7EEDBFF1CDC3

### DISCUSSION THREAD

**Discussion by giscus** [celestiamobile/comments#111] [General]
# E8D83035-EF68-4E18-AB78-7EEDBFF1CDC3

41 Comae Berenices is a binary star system of a K-type giant star and a red dwarf about 379.46 light-years away in the constellation Coma Berenices. The larger component star (A) has a superjovian planet about six times more massive than Jupiter, which completes a single orbit once every 1.67 years

https://celestia.mobi/addon?item=E8D83035-EF68-4E18-AB78-7EEDBFF1CDC3

**Comment by EricNelson-sketch** [comment]
Very realistic for its colors.
